src/content/faq/General.md

```diff
@@ -8,20 +8,76 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
 
 ## Why should I use GraphQL?
 
+It depends on your use case, but in general, GraphQL has a few key features that stand out. For example, GraphQL enables you to:
+
+* Improve performance and stability by [sending queries](/learn/queries/) for the exact data you need. 
+* Aggregate data from [multiple UI components](/learn/queries/#fragments) in [a single request](/learn/best-practices/#server-side-batching-caching ).
+* Create a representation of your data that feels familiar and natural ([a graph](/learn/thinking-in-graphs/#it-s-graphs-all-the-way-down-https-en-wikipedia-org-wiki-turtles-all-the-way-down)).
+* Ensure that all of your data is [statically typed](/learn/schema/) and these types inform [what queries the schema supports](/learn/introspection/).
+* [Reduce the need for breaking changes](/learn/best-practices/#versioning), but utilize a [built-in mechanism for deprecations](https://spec.graphql.org/draft/#sec-Deprecation) when you need to.
+* Examine queries and responses through community tools like [GraphiQL](https://github.com/graphql/graphiql).
+
+And many others! [Our homepage](/) outlines some more convincing reasons to use GraphQL.
+
+Additionally, it doesn’t take rewriting your entire application to try out GraphQL. You can even start with a single HTTP request that wraps an existing REST call, for instance. Your [GraphQL schema](/learn/thinking-in-graphs/#shared-language) and [business domain model](/learn/thinking-in-graphs/#business-logic-layer) can be expanded gradually, so we’d recommend focusing on one use case at first and only build the part of the schema you need for that to work. 
+
+
 ## Is GraphQL a database language like SQL?
 
+No, but this is a common misconception.
+
+GraphQL is a specification for how to request and modify data. That data can be provided by [arbitrary functions](/learn/) and those functions can call APIs, databases, [local cache](/learn/caching/) or any representation of data. The main and most common use case is calling APIs.
```

Comment:
Databases are almost always exposed via some form of API, so I am not sure this differentiation makes sense. 
Based on that I am also not sure I agree with `main and most common use case is calling APIs`: Getting data from a database is very common, maybe the most common use case in my experience.

Reply:
I see what you mean. The API use case is the one that is most highlighted on the homepage and promoted in other internal documents, so that's why I felt like it was important to distinguish here.

I'd be curious to hear how you'd distinguish GraphQL from SQL - mostly because then I can incorporate that into this answer 😂

Reply:
I am not sure I have a great answer to SQL vs GraphQL, but I think there are two fundamental aspects if you really want to compare both:

1. Where the data comes from: GraphQL is agnostic to the actual datasource: GraphQL is a data aggregator across (possibly) multiple arbitrary and therefore heterogeneous data sources. SQL is a query language for a single relational database.  

2. "Distance" between client and server or visibility of the API: GraphQL is a query language between a client and server where the client is normally  "far" away. The client is normally a web/mobile/native client or the client can be even arbitrary for public GraphQL APIs. Compare that with SQL: normally SQL is an implementation detail of a backend server and not in any way exposed to a frontend or to the public.  

These are general statements describing the most common and best practices: there are exceptions to nearly everything said above (e.g. some are using SQL statements directly in the frontend, some are using GraphQL to access their database instead of SQL etc).

I hope that helps. :-)

Reply:
I'm not sure what I've written here is the best FAQ answer (also it's a bit long), but maybe it'll help inspire something better?

---

No, but this is a common misconception.

GraphQL is a specification typically used for remote client-server communications, and is agnostic to the data source(s) used to retrieve data/persist changes. Data access/manipulation is performed via [arbitrary functions](/learn/) called "resolvers"; GraphQL coordinates and aggregates the data from these resolver functions, returning the result to the client. General advice is that these resolver functions should delegate to a "business logic layer", written in your organizations' preferred programming language, responsible for communicating with the various underlying data sources (remote APIs, databases, [local cache](/learn/caching/), and nearly anything else your programming language can access).

Reply:
Both of these are _super_ helpful, thank you @andimarek and @benjie! I'm going to take both and hopefully come up with an answer that covers everything (but isn't too long).

Reply:
Alright I tried, curious what you think 😁